internal/models/makemetric.go

```diff
@@ -123,11 +122,9 @@ func makemetric(
 		case float64:
 			// NaNs are invalid values in influxdb, skip measurement
 			if math.IsNaN(val) || math.IsInf(val, 0) {
-				if debug {
```

Comment:
This should have been caught in an earlier change. The `debug` flag has been removed in favor of the `D!` log prefix